tools/dockerfile/interoptest/grpc_interop_python/Dockerfile

```diff
@@ -73,6 +73,14 @@ RUN apt-get update && apt-get install -y \
     python3-all-dev \
     python-pip
 
+# Install Pyenv and associated Python versions
+RUN curl -L https://raw.githubusercontent.com/yyuu/pyenv-installer/master/bin/pyenv-installer | sh
+RUN pyenv update
+RUN pyenv install 2.7-dev
```

Comment:
actually, I think we are using tox for selecting python versions. Does python installed here make any difference for actually running the tests? I don't think so.